src/App.js

```diff
@@ -48,6 +49,14 @@ function App() {
               </AuthWrapper>
             )}
           />
+          <Route
+            path="/admins"
```

Comment:
hmm for some reason `/admins` is giving me a 404 error
<img width="1287" alt="image" src="https://github.com/FoodIsLifeBGP/banana-admin/assets/13320440/58fb499c-7f6d-41d4-b152-583990bd266f">

Reply:
Something seems amiss here...that's not the correct styling for the 404 page. The nav doesn't look right either

Reply:
@tackc oh crap i'm a fool, i checked out the wrong branch! hang on sorry

Reply:
works as described sorry, i had a long day